Examples/Sources/ChatbotExample/Views/ChatSettingsDialog.swift

```diff
@@ -0,0 +1,466 @@
+                    .cornerRadius(4)
+                }
+            } else if model.availableModels.isEmpty {
+                VStack(spacing: 8) {
+                    // Fallback to default models if API call fails
+                    LLMOptionButton(
+                        title: "GPT-3.5 Turbo",
+                        subtitle: "Fast and efficient",
+                        model: .gpt3_5Turbo,
+                        selectedModel: $model.selectedModel
+                    )
+                    
+                    LLMOptionButton(
+                        title: "GPT-4",
+                        subtitle: "Most capable",
+                        model: .gpt4,
+                        selectedModel: $model.selectedModel
+                    )
+                    
+                    LLMOptionButton(
+                        title: "GPT-4 Turbo",
+                        subtitle: "Latest and fastest GPT-4",
+                        model: .gpt4_turbo,
+                        selectedModel: $model.selectedModel
+                    )
+                }
+            } else {
+                VStack(spacing: 8) {
+                    ForEach(model.availableModels) { modelInfo in
+                        DynamicLLMOptionButton(
+                            modelInfo: modelInfo,
+                            selectedModel: $model.selectedModel
+                        )
+                    }
+                }
+            }
+        }
+    }
+    
+    // MARK: - Temperature Section
+    
+    private var temperatureSection: some View {
+        VStack(alignment: .leading, spacing: 12) {
+            Text("Temperature")
+                .font(.headline)
+                .fontWeight(.semibold)
+            
+            VStack(spacing: 8) {
+                HStack {
+                    Text("More focused")
+                        .font(.caption)
+                        .foregroundColor(.gray)
+                    
+                    Spacer()
+                    
+                    Text("\(String(format: "%.1f", model.temperature))")
+                        .font(.caption)
+                        .fontWeight(.medium)
+                        .foregroundColor(AppColors.primary)
+                        .padding(4)
+                    
+                    Spacer()
+                    
+                    Text("More creative")
+                        .font(.caption)
+                        .foregroundColor(.gray)
+                }
+                
+                // Temperature slider
+                Slider($model.temperature, minimum: 0.0, maximum: 1.0)
+            }
+        }
+    }
+    
+    // MARK: - Max Tokens Section
+    
+    private var maxTokensSection: some View {
+        VStack(alignment: .leading, spacing: 12) {
+            Text("Max Tokens")
+                .font(.headline)
+                .fontWeight(.semibold)
+            
+            TextField("1000", text: $model.maxTokens)
```

Comment:
I don't believe that maxTokens is actually used anywhere, in which case it can just be removed. Correct me if I'm wrong